python/tvm/script/tir/__init__.pyi

```diff
@@ -211,8 +211,12 @@ def alloc_buffer(
 special_stmt - Reads/Writes
 """
 
-def reads(read_regions: Union[BufferSlice, List[BufferSlice]]) -> None: ...
-def writes(write_region: Union[BufferSlice, List[BufferSlice]]) -> None: ...
+def reads(
+    read_regions: Union[BufferSlice, List[BufferSlice]], *other_regions: BufferSlice
+) -> None: ...
+def writes(
+    write_region: Union[BufferSlice, List[BufferSlice]], *other_regions: BufferSlice
+) -> None: ...
```

Comment:
Just curious, is it possible to use @overload to separate the logic more clearly?

Reply:
using @overload in TIR script will cause an error as @overload will call an anonymous function which is not allowed currently [here](https://github.com/apache/tvm/blob/3f1c931aa1414b69103c8c3ca019a9f55b5cb7a7/python/tvm/script/utils.py#L47) in TVM.

Reply:
done